### PR TITLE
Update tool development slides

### PR DESCRIPTION
**Pull request by abretaud**
Preparing for Roscoff
Mostly small changes.
I've moved a few slides that were drowned in the toolshed slide into the tool-integration slides. This way tool-integration is self-sufficient most of the time, and if people want to publish to the toolshed manually, they can look at the toolshed slides.

I'm reviewing the conda slides at the moment

### CODE REVIEW

**Comment by abretaud** [comment]
Ok, finished the conda part, it's ready for a review I think

File: topics/dev/tutorials/tool-integration/slides.html [hunk 1]
```diff
@@ -1179,20 +1199,24 @@
 
 ### Dataset collection
 
-Creation when uploading (>=17.05)
+Creation when uploading
+
+![collectionPrefiltered](../../../galaxy-data-manipulation/images/collectionPrefiltered.png)
 
-![](../../images/upload_collection.png)
+Also possible from rule-based uploader
```
Comment (bgruening) [inline topics/dev/tutorials/tool-integration/slides.html]: link to the training of the rbu?

File: topics/dev/tutorials/tool-integration/slides.html [hunk 2]
```diff
@@ -1289,6 +1313,69 @@
...
+Many tools developed by the community on GitHub repositories
+- [Intergalactic Utilities Commission](https://github.com/galaxyproject/tools-iuc)
+- [GalaxyP](https://github.com/galaxyproteomics/tools-galaxyp)
+- ...
+
+Added value:
+- Easier development for developers
+- Easier contribution for user
+- Automated tests on each contribution
+- Automated publishing to ToolShed
```
Comment (bgruening) [inline topics/dev/tutorials/tool-integration/slides.html]: Principle of many eyes.

File: topics/dev/tutorials/tool-integration/slides.html [hunk 3]
```diff
@@ -1311,6 +1408,73 @@
 
 ---
 
+### ToolShed
+
+- Need to create a `.shed.yml` file in the current repository (tool repository):
```
Comment (nsoranzo) [inline topics/dev/tutorials/tool-integration/slides.html]:
```suggestion
- Need to create a `.shed.yml` file in the tool directory of the GitHub repository:
```

File: topics/dev/tutorials/tool-integration/slides.html [hunk 4]
```diff
@@ -1311,6 +1408,73 @@
...
+                     --category="Sequence Analysis"
+                     [--remote_repository_url=<URL to .shed.yml on github>]
+                     [--homepage_url=<Homepage for tool.>]
+```
+
+---
+
+### Tool suites
+
+- Tools and tool suite `.shed.yml`: implicitly define repositories for each individual tool in the directory and build a suite for those tools.
```
Comment (nsoranzo) [inline topics/dev/tutorials/tool-integration/slides.html]: Need to explain what a suite is/does?

File: topics/dev/tutorials/tool-integration/slides.html [hunk 5]
```diff
@@ -1179,20 +1199,24 @@
...
-![](../../images/upload_collection.png)
+Also possible from rule-based uploader
 
 ---
 
 ### Dataset collection > input
 
 ```xml
-<param name="inputs" type="data" format="bam" multiple="true" label="Input BAM(s)" />
+<param name="inputs" type="data" format="bam" label="Input BAM(s)" />
```
[the remark this answers is not in the document]
Comment (abretaud) [reply topics/dev/tutorials/tool-integration/slides.html]: Yep, but we explain a few slides before the difference with/without `multiple="true"`, that's the same behavior for data collections?
The screenshot was done without multiple=true I think, that's why I changed it

File: topics/dev/tutorials/tool-integration/slides.html [hunk 6]
```diff
@@ -1149,7 +1170,7 @@
 
 ### Dataset collection
 
-To group related datasets, and treat them as a single entity
+A dataset collection combines numerous datasets in a single entity that can be manipulatedtogether
```
Comment (nsoranzo) [inline topics/dev/tutorials/tool-integration/slides.html]:
```suggestion
A dataset collection combines numerous datasets in a single entity that can be manipulated together
```

Comment (abretaud) [reply topics/dev/tutorials/tool-integration/slides.html]: that was fast :) thanks

**Comment by bgruening** [comment]
Merci!

**Pull request merged**